ingest/lambdas/index-rectifier.js

```diff
@@ -0,0 +1,28 @@
+// @ts-check
+const ir = require('../lib/index-rectifier');
+
+const handler = async () => {
+  const region = process.env.AWS_REGION;
+
+  // Perform an Index and DSpace diff.
+  const result = await ir.diff(
```

Comment:
This function only takes 2 arguments but you're passing it 3.

Reply:
Fixed. I had ts-check on and missed that error.